app/src/engineering/java/com/hedvig/app/DevelopmentActivity.kt

```diff
@@ -40,6 +43,10 @@ class DevelopmentActivity : AppCompatActivity(R.layout.activity_development) {
             it.submitList(
                 listOf(
                     DevelopmentScreenAdapter.DevelopmentScreenItem.Header,
+                    DevelopmentScreenAdapter.DevelopmentScreenItem.Row("Embark Story Tester") {
+                        loadKoinModules(embarkStoryTesterModule)
```

Comment:
This will crash if this row is pressed twice.

I suggest adding this loading to `EmbarkStoryTesterActivity::onCreate`, and pairing it with an `unloadKoinModules` in `EmbarkStoryTesterActivity::onDestroy`.